src/portal-components/DetailsPage/utils.ts

```diff
@@ -6,10 +6,19 @@ import { scrollToWithOffset } from 'utils'
 export function getComponentId(
   rowSynapseConfig: RowSynapseConfig,
   entityTitle: string = '',
-  index: number = 0,
```

Comment:
I decided to drop the index. If we have a collision, then we should update the config titles to not collide.